packages/server/src/fhir/operations/export.test.ts

```diff
@@ -191,9 +190,7 @@ describe('System export', () => {
     expect(dataRes.status).toBe(200);
 
     // Output format is "ndjson", new line delimited JSON
-    // However, we only expect one Observation, so we can parse it as JSON
     const resourceJSON = dataRes.text.trim().split('\n');
-    expect(resourceJSON).toHaveLength(1);
```

Comment:
The assertion that was failing earlier today. Looks like duplicate observations are being created similar to another export flaky test:
https://github.com/medplum/medplum/pull/2022#discussion_r1191719103